### ISSUE TITLE
captcha image file not found

### ISSUE THREAD

**Issue by vlfig-iov42** [cevoaustralia/aws-google-auth#240]
Seems that upgrading `python-pillow` may break the captcha flow.

Sample output:
```
Please visit the following URL to view your CAPTCHA: https://accounts.google.com/Captcha?v=2&ctoken=[snip]
Captcha (case insensitive): kf.service.services: KApplicationTrader: mimeType "x-scheme-handler/file" not found
org.kde.kdegraphics.gwenview.lib: Unresolved mime type  "image/x-mng"
org.kde.kdegraphics.gwenview.lib: Unresolved raw mime type  "image/x-samsung-srw"

Invalid username or password
```
While the image viewer program says that it can't find the file.

There's an upstream ticket https://github.com/python-pillow/Pillow/pull/5945 with a fix ~https://github.com/python-pillow/Pillow/pull/5950~ https://github.com/python-pillow/Pillow/pull/6010.

A solution would be to either downgrade `python-pillow` to 8.4.x or wait for that fix in some subsequent 9.x version.

There are no changes to be made to `aws-google-auth`. This is just for tracking purposes.

**Comment by hugovk** [comment]
https://github.com/python-pillow/Pillow/pull/6010 has been released in Pillow 9.0.1.

https://github.com/python-pillow/Pillow/releases/tag/9.0.1

**Comment by vlfig-iov42** [comment]
Confirm fixed with 9.0.1.
